c/dsutils.c

```diff
@@ -0,0 +1,417 @@
+#ifdef METTLE
+/* HAS NOT BEEN COMPILED WITH METTLE BEFORE */
+#else
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdarg.h>
+#include <sys/stat.h>
+#include "zowetypes.h"
+#include "alloc.h"
+#include "utils.h"
+#include "json.h"
+#include "bpxnet.h"
+#include "logging.h"
+#include "unixfile.h"
+#ifdef __ZOWE_OS_ZOS
+#include "zos.h"
+#endif
+
+#include "charsets.h"
+
+#include "socketmgmt.h"
+#include "httpserver.h"
+#include "datasetjson.h"
+#include "pdsutil.h"
+#include "jcsi.h"
+#include "impersonation.h"
+#include "dynalloc.h"
+#include "utils.h"
+#include "vsam.h"
+#include "qsam.h"
+#include "icsf.h"
+
+
+
+#include "dsutils.h"
+
+
+#ifdef __ZOWE_OS_ZOS
+/*
+  Dataset only, no wildcards or pds members accepted - beware that this should be wrapped with authentication
+ */
+int getVolserForDataset(const DatasetName *dataset, Volser *volser) {
```

Comment:
Is the content of this file the same as the one you removed from `datasetjson.c`?

Reply:
Yes, it's the same except that I removed the 'static' in front of the functions.